libs/core/src/lib/table/directives/table-cell.directive.ts

```diff
@@ -14,6 +14,10 @@ import { CheckboxComponent } from '@fundamental-ngx/core/checkbox';
     selector: '[fdTableCell], [fd-table-cell]'
 })
 export class TableCellDirective implements AfterContentInit {
+    
+    @HostBinding('attr.aria-label')
+    @Input()
+    ariaLabel: string = null;
```

Comment:
It's the directive, so we can set `aria-label` on the host element, i don't understand why we should have binding like that, could you please explain?